core/controlplane/config/templates/cloud-config-worker

```diff
@@ -634,9 +634,9 @@ write_files:
              mv -f $f ${encKey%.enc}
            done;
 
-           {{ if .Experimental.TLSBootstrap.Enabled }}
+           {{ if and .Experimental.TLSBootstrap.Enabled .AssetsConfig.HasTLSBootstrapToken }}
            echo injecting token into the kubelet bootstrap kubeconfig file
-           bootstrap_token=$(cat /etc/kubernetes/auth/kubelet-bootstrap.token);
+           bootstrap_token=$(cat /etc/kubernetes/auth/tls-bootstrap.token);
```

Comment:
Just for my education but why did you rename it from kubelet- to tls-?
I believe you tried to match it against other relevant symbol names(TLSBootstrap, HasTLSBootstrapToken).
However, the naming in kubelet is `kubelet-bootstrap` rather than `tls-bootstrap`.
I guess you assumed (like me) that it is originally meant for `kubelet-(tls-)bootstrap` hence named `tls-bootstrap`?